src/simple/room/Room.js

```diff
@@ -167,6 +169,7 @@ class Room extends Emitter {
     Wazo.Phone.off(this.ON_CHAT, this._boundOnChat);
     Wazo.Phone.off(this.ON_SIGNAL, this._boundOnSignal);
     Wazo.Phone.off(this.ON_SCREEN_SHARE_ENDED, this._boundOnScreenshareEnded);
+    Wazo.Phone.off(this.ON_VIDEO_INPUT_CHANGE, this._saveLocalVideoStream.bind(this));
```

Comment:
`bind(this)` create another  function.
To be able to unregister an event you have to :
```js
const boundSaveLocallVideoStream = this._saveLocalVideoStream.bind(this);
Wazo.Phone.off(this.ON_VIDEO_INPUT_CHANGE, boundSaveLocallVideoStream);
/// ...
Wazo.Phone.on(this.ON_VIDEO_INPUT_CHANGE, boundSaveLocallVideoStream);
```

Reply:
Makes total sense once you spent more than 10 seconds reading the code... thank you, Master